.github/workflows/prestocpp-worker-with-clp-connector-runtime-image-build.yml

```
@@ -0,0 +1,64 @@
+        uses: "docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4"
+        with:
+          context: "./presto-native-execution/"
+          file: "./presto-native-execution/scripts/dockerfiles/ubuntu-22.04-dependency.dockerfile"
+          push: >-
+            ${{github.event_name != 'pull_request'
+            && github.ref == 'refs/heads/release-0.293-clp-connector'}}
+          tags: "${{steps.meta.outputs.tags}}"
+          labels: "${{steps.meta.outputs.labels}}"
+
+      - name: "Update Metadata for Runtime Image"
+        id: "meta"
```

actionlint (1.7.7)

step ID "meta" duplicates. previously defined at line:25,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)